app/src/main/java/com/kickstarter/viewmodels/CommentsViewModel.java

```diff
@@ -47,6 +49,13 @@ public CommentsViewModel(final @NonNull Environment environment) {
     this.client = environment.apiClient();
     this.currentUser = environment.currentUser();
 
+    final Observable<User> currentUser = Observable.merge(
```

Comment:
ehhh not super happy with this current user, think i may take a pass again tmrw a.m. to walk away from using `currentUser.observable()` like this